package/src/contexts/messageContext/MessageContext.tsx

```diff
@@ -92,7 +88,7 @@ export type MessageContextValue<
   onPressIn: ((payload: TouchableHandlerPayload) => void) | null;
   /** The images attached to a message */
   otherAttachments: Attachment<StreamChatGenerics>[];
-  reactions: Reactions;
+  reactions: ReactionSummary[];
```

Comment:
this is a breaking change in my view

Reply:
for those who have custom reactions UI implemented

Reply:
can we support both.. with marking the previous as deperecated?

Reply:
I have made the fields optional.